aireview/src/aireview/domain/services/code_analyzer.py

```diff
@@ -32,4 +33,8 @@ def _build_analysis_prompt(self, file: PullRequestFile):
 
 
     def _parse_analysis(self, analysis):
```

Comment:
**style**: Type hint [ReviewComment] should use List from typing module for better static type checking.